catalogue/webapp/pages/works.js

```diff
@@ -330,6 +332,8 @@ const Works = ({ works, searchParams }: Props) => {
                               page,
                               workType,
                               _queryType,
+                              [`production.dates.from`]: productionDatesFrom,
+                              [`production.dates.to`]: productionDatesTo,
```

Comment:
What's the need for the square brackets and backticks? Could this be

```
'production.dates.from': productionDatesFrom,
'production.dates.to': productionDatesTo,
```

?

Reply:
I want to store the information on the reporting index as the same as the API / URL parameters.

> Might have another think about how we do this and tying it to the URL (hence I've used the URL param names).

Reply:
Stupid me, just re-read, and get your point.

Reply:
Fixed